lib/rbi/printer.rb

```diff
@@ -466,6 +469,15 @@ def visit_sig(node)
       end
     end
 
+    # @override
+    #: (RBSSig node) -> void
+    def visit_rbs_sig(node)
+      print_loc(node)
+      printt("#")
```

Comment:
Let's strip the `:` when we create the sig so it's nicer to handle, then we add it back when we print:

```suggestion
      printt("#:")
```